cluster/manifests/roles/poweruser-role.yaml

```diff
@@ -838,3 +838,18 @@ rules:
   - create
   - update
   - patch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  - rolebindings
+  - clusterroles
+  - clusterrolebindings
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
In which case do you need the delete those resources as a poweruser?

Reply:
Deleting would be needed if you want to cleanup resources that you no longer need. e.g. you created a `clusterrole` to try out an operator, but then later decide to remove it.

I thought about also removing `delete` but if you can `update`/`patch` you can cause the same "harm" anyway so I don't see that it brings much value to prevent `delete`.

Reply:
thanks, feel free to merge

Reply:
cleanup your deployment,svc,ingress if you move to a different namespace, f.e.